Onboarding/Network requirements

```diff
@@ -0,0 +1,104 @@
+Network requirements for IP allowlist:
+
+Defender for Cloud services use the following list of IPs to monitor the security status of customer environments. 
+The list is applicable to all Defender plans and full capabilities of the CSPM Foundational (free) offering.
```

Comment:
Need to remove this sentence as it's apparently not correct. the sentence should be:
This list is applicable to the CSPM Foundational (free) offering.